src/styles/index.css

```diff
@@ -0,0 +1,228 @@
+  color: #000;
+  text-align: center;
+}
+
+.nav__list-link:hover {
+  color: var(--accent-color);
+}
+
+.is-active {
+  color: var(--accent-color);
+}
+
+.is-active::after {
+  content: '';
+
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  background-color: var(--accent-color);
+  border-radius: 8px;
+  right: 0;
+  bottom: 0;
+}
+
+/* #endregion */
+
+.main{
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+  justify-items: center;
+}
+
+/* #region card Styles */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+
+.card {
+  width: 200px;
+  border: 1px solid var(--border-color);
+  padding: 32px 16px 16px;
+  font-weight: 400;
+  background-color: var(--white);
+  color: var(--accent-text-color);
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  height: 134px;
+  width: 160px;
+}
+
+.card__title{
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-top: 40px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  margin-bottom: 24px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__rate{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  margin: 0;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-top: 2px;
+  color: var(--text-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-title {
+  font-size: 12px;
+  line-height: 12px;
+  color: var(--main-text-color);
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  width: 100%;
+  height: 40px;
+  background-color: var(--accent-color);
+  font-weight: 700;
+  font-size: 14px;
+  border: 1px solid var(--white);
+  border-radius: 5px;
+  line-height: 1.14px;
+  color: var(--white);
+  text-transform: uppercase;
+  text-decoration: none;
+  transition:
+  background-color 0.3s,
+  color 0.3s;
+}
+
+.card__button:hover {
+  background-color: var(--accent-color);
+  color: var(--accent-color);
+  border-color: var(--accent-color);
+  cursor: pointer;
```

Comment:
The hover state for '.card__button' is not changing anything because the background-color, color, and border-color are set to the same values as in the default state. Adjust the hover state to create a visible effect when a user hovers over the button.